src/amo/actions/reviews.js

```diff
@@ -55,7 +55,7 @@ export const UPDATE_RATING_COUNTS: 'UPDATE_RATING_COUNTS' =
 export type ReviewAddonType = {|
   iconUrl: string,
   id: number,
-  name: string,
+  name: Object,
```

Comment:
Why this change?

Reply:
Without this change, the Flow tests fail:

<img width="978" alt="Screenshot 2023-09-26 at 12 41 23 PM" src="https://github.com/mozilla/addons-frontend/assets/63154973/0a51ea32-f076-4ba0-9d05-5cdd926f277f">

Please let me know if there's a better fix!

Reply:
that's happening because of your changes on lines 363 to 366, that I left the other comment about.